Editor/InputSystemChecker.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using Innoactive.Creator.Core.Input;
+using UnityEditor;
+
+[InitializeOnLoad]
+public static class InputSystemChecker
+{
+    private const string message =
+        "From Creator version 2.11 onwards Unity's new Input System is required but not added to your project yet." +
```

Comment:
```suggestion
        "From Creator version 2.11 onwards Unity's new Input System is required." +
```

If we say it is not added, it just raises the question why not ;-)